frontend/src/lib/components/CommandBar/ActionInput.tsx

```diff
@@ -17,9 +17,8 @@ type PrefixIconProps = {
 const PrefixIcon = ({ activeFlow }: PrefixIconProps): React.ReactElement | null => {
     if (activeFlow) {
         return <activeFlow.icon className="palette__icon" /> ?? <IconPencil className="palette__icon" />
-    } else {
-        return <IconChevronRight className="palette__icon" />
     }
+    return <IconChevronRight className="palette__icon" />
```

Comment:
There are a couple of unrelated changes like this due to running `eslint` / `prettier` after using VS Code's refactor.